Helper/Setup.php

```diff
@@ -141,11 +141,6 @@ public function upgradeData(string $mode, ModuleDataSetupInterface $setup, Modul
                 $moduleVersion = $moduleData['setup_version'] ?? '';
                 $storeEmail = $this->scopeConfig->getValue('trans_email/ident_sales/email') ?? '';
                 $storeUrl = rtrim($this->storeManager->getStore()->getBaseUrl(UrlInterface::URL_TYPE_WEB, true), '/');
-                if (stripos($storeUrl, self::MAGENTO_REQUIRED_PROTOCOL) !== 0) {
```

Comment:
Should the unused `MAGENTO_REQUIRED_PROTOCOL` const be removed from line 35 as well?

Reply:
@rob-ns8 Absolutely, I my commits got messed up after switching base branches. Will re-add it manually now

Reply:
Addressed in 4f8b4cc3d42e3723fd532a5e577ec3759713093f